src/demos/AirshipModal.tsx

```diff
@@ -10,9 +10,9 @@ import {
 
 import { AirshipBridge } from '../Airship'
 import { KeyboardTracker } from './KeyboardTracker'
-import { LayoutContext, SafeAreaGap } from './LayoutContext'
-
-type ChildFunction = (gap: SafeAreaGap) => React.ReactNode
+import { EdgeInsets } from 'react-native-safe-area-context'
+import { SafeAreaConsumer } from 'react-native-safe-area-context'
+type ChildFunction = (gap: EdgeInsets) => React.ReactNode
```

Comment:
The spacing seems wrong here, and in a few other places. How did you get past the pre-commit hooks? Please run `yarn lint` and / or `yarn fix` to clean this up.